pkg/template/template.go

```diff
@@ -222,6 +242,16 @@ func (t *dirTemplate) Execute(dirPrefix string) error {
 	})
 }
 
+func isBinary(r io.Reader) (bool, error) {
+	buf := make([]byte, 1024)
+	n, err := r.Read(buf)
+	if err != nil && err != io.EOF {
+		return false, err
+	}
+
+	return !utf8.Valid(buf[:n]) || bytes.ContainsAny(buf[:n], "\x00"), nil
```

Comment:
Checking on `NULL` enough or can there be more octets added? I never added binaries to my boilr templates my self.

Reply:
Yeah I know ... but I thougt maybe the second or could be extended... nvm :) Looks good.

Reply:
Let's decide what buffer size to set?
I decided to limit it, because who knows what people will decide to add to their templates, maybe it will be 4K video :)

Reply:
Not sure... did a quick search for how to best check for binary files in go. I saw this mimetype library that seems to suggest that the max size of a file header is 262 bytes... so that will mean the first 1024 will contain atleast some of the binary data (if it is). https://github.com/h2non/filetype

BTW...  Don't really have time to test it (extensively) so I will be relying on you for the testing (of both of your PRs). The code seems good as far as I can see. So just let me know if you think it's ready to be merged.

Reply:
I found a bug in this RP, wait for a fix

Reply:
I think they are now ready for merge.